models/Events.js

```diff
@@ -0,0 +1,36 @@
+import { Schema, model, models } from 'mongoose';
+
+const EventsSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  data_source: {
+    type: String,
+    required: true,
+  },
+  agreement: {
+    type: Number,
+    required: true,
+  },
+  status: {
+    type: String,
+    enum: ['Si cumple', 'No cumple', 'Alto riesgo'],
```

Comment:
De repente podríamos tener el enum con valores en inglés, para mantener la consistencia con el código:
```suggestion
    enum: ['yes', 'no', 'risk'],
```